src/MandrillTransport.ts

```diff
@@ -8,6 +10,20 @@ export interface Options {
   apiKey: string;
 }
 
+export type EmailStatus = {
```

Comment:
**suggestion**: rename it to MandrillStatus or MandrillResponse, to better understand that this type refers specifically to the Mandrill code.

Reply:
done